db/models/models.js

```diff
@@ -59,4 +59,12 @@ exports.fetchAllUsers = () => {
     }
     return results.rows;
   });
-};
\ No newline at end of file
+};
+
+exports.insertComment = (username, body, id) => { 
+    return db.query(`SELECT articles.article_id FROM articles WHERE article_id = ${id}`).then(()=>{
```

Comment:
I dont know if you need to do this query here, I'm assuming its so that you can get the correct errors for invalid/non-existent article_ids but your kind of creating some blocking code here, it might be worth having a look at it at and seeing if it could be refactored